src/src/StringInformationSignal.java

```diff
@@ -0,0 +1,25 @@
+public class StringInformationSignal implements informationSignal<String>{
+    private String value;
```

Comment:
```suggestion
public class StringInformationSignal implements informationSignal<String>{

    private String value;
```
add linebreaks at the beginning + end of the class